test/test_api/test_api.py

```diff
@@ -405,29 +406,31 @@ def test_tabular_regression(openml_name, resampling_strategy, backend, resamplin
     assert 'Estimator' in representation
 
 
-@unittest.mock.patch('autoPyTorch.evaluation.train_evaluator.eval_train_function', new=dummy_eval_train_function)
-@pytest.mark.parametrize('forecasting_toy_dataset', ['multi_variant_wo_missing'], indirect=True)
+@pytest.mark.parametrize('forecasting_toy_dataset', ['uni_variant_wo_missing'], indirect=True)
 @pytest.mark.parametrize('resampling_strategy,resampling_strategy_args',
                          ((HoldoutValTypes.time_series_hold_out_validation, None),
-                          (CrossValTypes.time_series_cross_validation, {'num_splits': CV_NUM_SPLITS}),
```

Comment:
if the tests are not passing due to the time limit, you can consider lowering the num splits. Also, the patch will help with that.

Reply:
Same as above, num_splits is already set as 2 and cannot be set even lower